mantidproject/mantid
# About links can now be opened with Firefox on Ubuntu

**Description of work.**
Captures the `linkActivated` signals and uses MantidDesktopServices to open the links

**To test:**
- **This needs to be a packaged build!**
- Open the installed packaged build on Ubuntu with Firefox as default browser
- Go to Help > About
- Click all the links. They should all work

Fixes #26128



---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
